atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ProfilingKeyValueService.java

```diff
@@ -45,6 +48,8 @@
 import com.palantir.util.paging.TokenBackedBasicResultsPage;
 
 public class ProfilingKeyValueService implements KeyValueService {
+    public static final String SLOW_LOGGER_NAME = "Profiling KVS Slow-Operation Logger";
```

Comment:
I would avoid this variable here.  Also for ease of use I would not use loggers with spaces in the name as it might cause trouble for certain logger configurations.  Something like `kvs-slow-log` should suffice.  And then adding documentation to help people use the logger appropriately - http://palantir.github.io/atlasdb/html/configuration/logging.html